src/features/movies/MovieList/MovieTile/styled.js

```diff
@@ -60,6 +72,7 @@ export const Title = styled.h3`
 `;
 
 export const Year = styled.p`
```

Comment:
I think it would be better if this was a `<time>` element. It would be perfect if we passed date instead of year in MovieTile props. From there we could include datetime attribute. For example:
 `<time datetime="2020-01-01">2020</time>`

[time element documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

Reply:
okey i added this check it out

Reply:
Yes, this is what i meant, you should also change MovieTile prop name from "year" to "date" because it is what we are passing now.

Reply:
okey done